ucm2/USB-Audio/Realtek/ALC4080-HiFi.conf

```diff
@@ -0,0 +1,57 @@
+SectionDevice."Speaker" {
+	Comment "Speakers"
+	Value {
+		PlaybackChannels 8
+		PlaybackPriority 200
+		PlaybackPCM "hw:${CardId}"
+		JackControl "Speaker - Output Jack"
+		PlaybackMixerElem "Speaker"
+	}
+}
+
+SectionDevice."Headphones" {
+	Comment "Front Headphones"
```

Comment:
Need to test.

Reply:
Works on MSI MAG X570S Torpedo Max (tested)

Reply:
Tested and it works.